osiris.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: OSIRIS
+  become: true
+  hosts:
+    - osiris
+  vars:
+    # This controls which of the two backup restore strategies to use
+    # backup_is_zip == true -> Use zip file from S3
+    # backup_is_zip == false -> Use the latest restic (daily) snapshop from S3
+    backup_is_zip: true
+    zip_name: denbi.zip
+    # This setting will be applied after restoring a backup
+    # If no version change is desired, make sure this version matches that of the backup
+    osiris_version: "1.5.1"
+  vars_files:
+    - group_vars/osiris.yml
+    - secret_group_vars/osiris.yml
+  pre_tasks:
+    - name: "Install system packages needed for installation"
+      ansible.builtin.dnf:
+        name: "{{ item }}"
+        state: present
+      loop:
+        - python3-pip
+        # We need these for SELinux but they are installed by mongodb_selinux
+        # - policycoreutils-python-utils
+    - name: "Install system packages needed for installation"
+      ansible.builtin.command: "pip install pymongo"
+  collections:
+    - community.mongodb
+  roles:
+    - hostname
+    - geerlingguy.repo-epel
+    - hxr.admin-tools
+    - geerlingguy.apache
+    - geerlingguy.php
+    - geerlingguy.apache-php-fpm
+
+    - community.mongodb.mongodb_repository
+    - community.mongodb.mongodb_install
+    - community.mongodb.mongodb_linux
+    - community.mongodb.mongodb_selinux
+    - community.mongodb.mongodb_auth
+    - community.mongodb.mongodb_mongod
+    # Custom
+    - denbi.ssh-root-users
+    - denbi.ssh-no-root-with-password
+    - denbi.osiris
+    - denbi.fail2ban-sshd
```

Comment:
Do you disable password auth?

Reply:
Yes but only for `root`. Regular users (which we don't plan to have) will still be able to login with a password.

For root access my public key is injected. Do we need to expand that list of keys?

Reply:
I think in the Infrastructure PR, you added our keypair to the VM so we should have access.
My feeling would be to disable password auth for all users, but if you use fail2ban it should be fine